mirgecom/mpi.py

```diff
@@ -184,3 +184,21 @@ def wrapped_func(*args: Any, **kwargs: Any):
         debug_remote_on_single_rank(MPI.COMM_WORLD, 0, func, *args, **kwargs)
 
     return wrapped_func
+
+
+# Change this to `if 1:` in order to prepend the rank number to every message
+# printed with print().
+if 0:
```

Comment:
```suggestion
def enable_rank_labeled_print():
    """Add docstring here.
    """
```
(and hook into docs)

Reply:
Done, thanks!

Reply:
Even better. This is cool.